<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>joinGame</class>
 <widget class="QWidget" name="joinGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>801</width>
     <height>198</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1" columnstretch="0,1">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="matchIDLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">    color: white;
                                        background-color: rgb(165, 29, 45);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;</string>
        </property>
        <property name="text">
         <string>Match ID:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditMatchID">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(165, 29, 45);
                                        background-color: rgba(229, 165, 10, 0.95);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;
                                        border-color: rgb(165, 29, 45);
                                    </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Insert Match ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPlayer2name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">    color: white;
                                        background-color: rgb(165, 29, 45);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;</string>
        </property>
        <property name="text">
         <string>Player 2 name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPlayer2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(165, 29, 45);
                                        background-color: rgba(229, 165, 10, 0.95);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;
                                        border-color: rgb(165, 29, 45);
                                    </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter Player 2 name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelPlayer1name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">    color: white;
                                        background-color: rgb(165, 29, 45);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;</string>
        </property>
        <property name="text">
         <string>Player 1 name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditPlayer1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(165, 29, 45);
                                        background-color: rgba(229, 165, 10, 0.95);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;
                                        border-color: rgb(165, 29, 45);
                                    </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter Player 1 name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonPlay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">    color: white;
                                        background-color: rgb(165, 29, 45);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;</string>
        </property>
        <property name="text">
         <string>PLAY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonBack">
        <property name="styleSheet">
         <string notr="true">    color: white;
                                        background-color: rgb(165, 29, 45);
                                        padding: 10px;
                                        border-radius: 10px;
                                        margin: 5px;</string>
        </property>
        <property name="text">
         <string>BACK TO OPTIONS</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
